src/translations/pages/contact.ts

```diff
@@ -951,6 +971,11 @@ export const Contact = {
 
     lastName: {
       label: _('Etternavn', 'Lastname', 'Etternamn'),
+      labelOptional: _(
+        'Etternavn (valgfritt)',
+        'Lastname (optional)',
```

Comment:
```suggestion
        'Last name (optional)',
```

Eventuelt "Surname"?